lib/scss/1_settings/_options.scss

```diff
@@ -151,5 +153,8 @@ $stereotype-add-submit-color: $color-magenta !default;
 $stereotype-add-input-border-radius: 17px !default;
 $stereotype-add-comment-color: $color-purple-dark !default;
 
-
+// Boards
+$board-menu-font-color: #042049 !default;
```

Comment:
Não use cores hardcoded porque isso obriga cada tema setar essa variável. Use uma das cores de papéis tipo $color-primary, $color-content-text, etc